### PR TITLE
auto deploy offical website

### PR DESCRIPTION
**Pull request by yetist**
I hope the official website can be automatically synchronized in real time:
1. If it is a PR or a commit on a non-master branch, deploy the compiled results to the preview site for review.
2. If there is a commit on the master branch, deploy the compiled results directly to the official website.

### CODE REVIEW

**Comment by yetist** [comment]
Now restart 184 on travis for test.

**Comment by yetist** [comment]
I think it's ready for merge, after merge, everything will be ok.

File: .travis.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ cache:
   - output/
 
 before_install:
-  - curl -Ls -o docker-build https://github.com/mate-desktop/mate-dev-scripts/raw/master/travis/docker-build
+  - curl -Ls -o docker-build https://github.com/mate-desktop/mate-dev-scripts/raw/travis/travis/docker-build
```
Comment (sc0w) [inline .travis.yml]: it doesn't need master branch?

Comment (yetist) [reply .travis.yml]: Yes, need master branch, but there has a PR block this. https://github.com/mate-desktop/mate-dev-scripts/pull/16

Comment (sc0w) [reply .travis.yml]: ah ok, so, needs to be merged other PR first?

**Pull request merged**